### ISSUE TITLE
React.HTMLProps showing up as undefined

### ISSUE THREAD

**Issue by ctaggart** [fable-compiler/ts2fable#42]
Affects `develop` branch for 0.2.0.

c:\Users\camer\fs\ts2fable\node_modules\@types\react\index.d.ts
``` ts
        interface IntrinsicElements {
            // HTML
            a: React.HTMLProps<HTMLAnchorElement>;
            abbr: React.HTMLProps<HTMLElement>;
            address: React.HTMLProps<HTMLElement>;
            area: React.HTMLProps<HTMLAreaElement>;
            article: React.HTMLProps<HTMLElement>;
            aside: React.HTMLProps<HTMLElement>;
```
``` fs
        type [<AllowNullLiteral>] IntrinsicElements =
            abstract a: undefined<HTMLAnchorElement> with get, set
            abstract abbr: undefined<HTMLElement> with get, set
            abstract address: undefined<HTMLElement> with get, set
            abstract area: undefined<HTMLAreaElement> with get, set
            abstract article: undefined<HTMLElement> with get, set
```

**Comment by ctaggart** [comment]
![image](https://user-images.githubusercontent.com/80104/32373358-d4174936-c090-11e7-8071-44a5a9bc2e28.png)

**Comment by ctaggart** [comment]
fixed by #47
